Dockerfile

```diff
@@ -18,5 +18,8 @@ WORKDIR /stripped_repo
 # Create gradle cache
 # `rw` option doesn't allow to make any changes on original dir but rather create something like overlayfs
 # We need this to allow `./gradlew` to write in `.gradle` directory
+# script for enforcing compatifility is runned for download IntelliJ dependencies
 RUN --mount=type=bind,rw,source=.,target=. \
   ./gradlew resolveDependencies
+
+RUN --mount=type=bind,rw,source=.,target=. ./gradlew buildPlugin && scripts/enforce-binary-compatibility
```

Comment:
Also, I think that now the `Verify binary compatibility with supported IntelliJ versions` step of the pipeline can be run unconditionally (not just for `master`) since it shouldn't (?) take much time + we have unlimited credit limit for a public repo

Reply:
Also, I think the two `RUN` instructions can be `&&`-ed together

Reply:
As agreed, it's better to leave this `RUN`s as is coz both layers are big enough, so this can speedup future modifications.